### PR TITLE
Adds automated builds and releases

### PR DESCRIPTION
**Pull request by bellerbrock**
## Description

_Provide a summary of your changes including motivation and context.
If these changes fix a bug or resolves a feature request, be sure to link to that issue._

closes https://github.com/FieldPheno/bryan-ellerbrock/issues/7

## Type of change

_What type of changes does your code introduce? Put an `x` in boxes that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [x] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Checklist:

- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated relevant documentation

## Changelog entry

_Please add a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note

```

### CODE REVIEW

**Comment by trife** [comment]
@bellerbrock Can you also add the Google Play upload section? You can leave it commented out for now, but want to get it in before merging everything.

File: .github/workflows/artifacts-cleanup.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        with:
+          ref: artifacts
+
+      - name: Commit removal of deleted branch's apk
+        uses: EndBug/add-and-commit@v9
+        with:
+          remove: "*${{ github.event.ref }}-Field-Book.apk"
+          message: "Remove artifacts linked to deleted ${{ github.event.ref }} branch"
+          push: --force
+          author_email: git-action-bot@example.com
```
Comment (chaneylc) [inline .github/workflows/artifacts-cleanup.yml:22]: Should this be a phenoapps email?

File: .github/workflows/github-release.yml [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+          echo "VERSION=$VERSION" >> $GITHUB_ENV
+          echo "RELEASE=$RELEASE" >> $GITHUB_ENV
+          echo "BUMP_TYPE=$BUMP_TYPE" >> $GITHUB_ENV
+
+      - name: Commit version changes
+        uses: EndBug/add-and-commit@v7
+        with:
+          add: 'version.properties'
+          message: Bump ${{ env.BUMP_TYPE }}
+          author_email: git-action-bot@example.com
```
Comment (chaneylc) [inline .github/workflows/github-release.yml:61]: Should this be a phenoapps email?

Comment (bellerbrock) [reply .github/workflows/github-release.yml:61]: I'm not sure what the best practice is, but yeah the existing email is just a placeholder so the actions are committed by a 'Git Action Bot'

Maybe we could make a PhenoApps Bot with a phenoapps email and use it for all the actions commits? The email could be supplied as a repository secret if it needs to be kept private.

**Review by chaneylc — APPROVED** [review]
Looks good. I wasn't able to test the whole process but I have a general idea of how the flow works and its documented well.

**Pull request merged**